repose-aggregator/core/repose-core/src/main/scala/org/openrepose/powerfilter/ReposeFilter.scala

```diff
@@ -116,21 +133,27 @@ class ReposeFilter @Inject()(@Value(ReposeSpringProperties.NODE.NODE_ID) nodeId:
 
         //Grab the traceGUID from the request if there is one, else create one
         val traceGUID = Option(wrappedRequest.getHeader(TRACE_GUID)) match {
-          case Some(string) if StringUtils.isNotBlank(string) => string
+          case Some(string) if StringUtils.isNotBlank(string) => TracingHeaderHelper.getTraceGuid(string)
           case _ => UUID.randomUUID.toString
         }
 
         MDC.put(TracingKey.TRACING_KEY, traceGUID)
 
-        val doFilterChain = TryWith(filterContextList) { _ =>
+        try {
+          // Ensure that the method name is supported
+          // todo: HTTP request methods are case-sensitive, so this check should not upper case the request method
+          if (!SupportedHttpMethods.contains(request.getMethod.toUpperCase)) {
```

Comment:
Was this added to the upgrade guide?

Reply:
Huh. So the reason I did not include this in those places is that I assumed it was already documented. I assumed that because if you look at current `master` you will notice that we already perform this check:
https://github.com/rackerlabs/repose/blob/master/repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilter.java#L415-L417

However, it appears that this check was added after the `8.10.0.0` release as part of the rewrite. So this change should have been documented as part of the release notes for **REP-3589**. So that is where I will add it now.